Date/MomentFormatConverter.php

```diff
@@ -59,6 +57,48 @@ class MomentFormatConverter
      */
     public function convert($format)
     {
-        return strtr($format, self::$formatConvertRules);
+        $size = strlen($format);
+        $foundOne = false;
+        $output = '';
+        //process the format string letter by letter
+        for ($i = 0; $i < $size; ++$i) {
+            //if finds a '
+            if ($format[$i] == "'") {
+                //if the next character are T' forming 'T', send a T to the
+                //output
+                if ($format[$i + 1] == 'T' && $format[$i + 2] == '\'') {
+                    $output .= 'T';
+                    $i += 2;
+                } else {
+                    //if it's no a 'T' then send whatever is inside the '' to
+                //the output, but send it inside [] (useful for cases like
+                //the brazilian translation that uses a 'de' in the date)
+                    $output .= '[';
+                    $temp = explode("'", substr($format, $i + 1))[0];
```

Comment:
But you can use current, like that : `$temp = current(explode("'", substr($format, $i + 1)));`